pandas/tests/window/moments/test_moments_consistency_rolling.py

```diff
@@ -136,6 +136,54 @@ def test_rolling_apply_consistency(
                 tm.assert_equal(rolling_f_result, rolling_apply_f_result)
 
 
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "window,min_periods,center", list(_rolling_consistency_cases())
+)
+def test_rolling_groupby(base_functions, window, min_periods, center):
+    base_df = DataFrame({"group": "A", "data": randn(20)})
+
+    b_df = base_df.copy()
+    b_df["group"] = "B"
+
+    grp_df = pd.concat([base_df, b_df]).groupby("group")
+
+    for (f, require_min_periods, name) in base_functions:
+        if (
+            require_min_periods
+            and (min_periods is not None)
+            and (min_periods < require_min_periods)
+        ):
+            continue
+
+        base_rolling_f = getattr(
+            base_df[["data"]].rolling(
+                window=window, center=center, min_periods=min_periods
+            ),
```

Comment:
how is this actually testing center? sure you pass it but unless we can see the expected results i don't have any idea whether this is correct. a small set of fixed cases that really show the input and output is much more useful here.

Reply:
Haven't digested the details here but it sounds like you should hard code expected results and test against that